From 7955ccd650dc3235df3bf6333222c2811dd43141 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Mon, 6 Jan 2014 10:27:23 -0500
Subject: [PATCH] libkeymap: include stdarg.h where used

Some of these headers use va_list but don't include stdarg.h for it.

Reported-by: Anthony Basile <blueness@gentoo.org>
Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 src/libkeymap/contextP.h      | 2 ++
 src/libkeymap/keymap/common.h | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/src/libkeymap/contextP.h b/src/libkeymap/contextP.h
index e3798e3..8ee3fab 100644
--- a/src/libkeymap/contextP.h
+++ b/src/libkeymap/contextP.h
@@ -1,6 +1,8 @@
 #ifndef LK_CONTEXTP_H
 #define LK_CONTEXTP_H
 
+#include <stdarg.h>
+
 #include "keymap.h"
 
 /**
diff --git a/src/libkeymap/keymap/common.h b/src/libkeymap/keymap/common.h
index 6b9cead..987b564 100644
--- a/src/libkeymap/keymap/common.h
+++ b/src/libkeymap/keymap/common.h
@@ -6,6 +6,8 @@
 #ifndef LK_COMMON_H
 #define LK_COMMON_H
 
+#include <stdarg.h>
+
 #include <keymap/context.h>
 
 /** Initializes the structures necessary to read and/or parse keymap.
-- 
1.8.4.3

